newIDE/app/src/SceneEditor/index.js

```diff
@@ -404,7 +444,10 @@ export default class SceneEditor extends Component {
         newName
       );
     }
-    done(true);
+    object.setName(newName);
+    if (done !== null) {
```

Comment:
You can remove this check for done !== null by always passing a function, even an empty one (`() => {}`) if you don't have anything to do.

Reply:
Can I make it optional like this?
_onRenameObject = (objectWithContext, newName, done = () => {}) => {
It will be easier to call it elsewhere that way